README.md

```diff
@@ -160,11 +160,11 @@ finder_icons = {
 -- you may need to increase this value
 finder_request_timeout = 1500,
 finder_action_keys = {
-    open = "o",
+    open = {"o", "<CR>"},
```

Comment:
```suggestion
    open = { 'o', '<CR>' },
```

Reply:
I've changed this and other entries as well for a consistent style of using quotes.